### PR TITLE
Add new relic, finalize db and data generation

### PR DESCRIPTION
**Pull request by timothydadams**
@JasperC1999 @jayrbarr

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -1,131 +1,130 @@
...
+    console.log(results);
+    cb(null, results);
   });
 };
 
-
 module.exports.create = create;
 module.exports.findId = findId;
 module.exports.updateId = updateId;
 module.exports.deleteId = deleteId;
```
Comment (jayrbarr) [inline db/index.js]: A cool shortcut for this would be:
`module.exports = { create, findId, updateId, deleteId };`

File: package-lock.json [hunk 2]
```diff
@@ -17301,8 +17507,7 @@
     "word-wrap": {
       "version": "1.2.3",
       "resolved": "https://registry.npmjs.org/word-wrap/-/word-wrap-1.2.3.tgz",
-      "integrity": "sha512-Hz/mrNwitNRh/HUAtM/VT/5VH+ygD6DV7mYKZAtHOrbs8U7lvPS6xf7EJKMF0uW1KJCl0H701g3ZGus+muE5vQ==",
-      "dev": true
+      "integrity": "sha512-Hz/mrNwitNRh/HUAtM/VT/5VH+ygD6DV7mYKZAtHOrbs8U7lvPS6xf7EJKMF0uW1KJCl0H701g3ZGus+muE5vQ=="
     },
     "worker-farm": {
       "version": "1.7.0",
```
Comment (jayrbarr) [inline package-lock.json:17513]: You could .gitignore package-lock.json but you probably will need to do a `git rm --cached` command now, if you want to do it.

**Review by jayrbarr — APPROVED** [review]
lgtm

File: db/index.js (hunk 1 above)
Comment (timothydadams) [reply db/index.js]: i agree.  its done :)

File: package-lock.json (hunk 2 above)
Comment (timothydadams) [reply package-lock.json:17513]: yea might be worth it!

**Pull request merged**